content-src/components/Base/Base.jsx

```diff
@@ -114,17 +116,29 @@ export class _Base extends React.PureComponent {
     const {initialized} = App;
 
     const prefs = props.Prefs.values;
-    if ((prefs.asrouterExperimentEnabled || prefs.asrouterOnboardingCohort > 0) && window.location.hash === "#asrouter") {
+
+    // Enable asrouter content
+    if (prefs.asrouterExperimentEnabled && !asrouterContent.initialized) {
+      asrouterContent.init();
+    } else if (!prefs.asrouterExperimentEnabled && asrouterContent.initialized) {
+      asrouterContent.uninit();
+    }
```

Comment:
Personally I am not a big fan of side effects happening in `render` functions; would I would probably do here is move this to `activity-stream.jsx` and do this logic in `store.subscribe`
```